src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -2432,9 +2369,6 @@ void GLHEBase::calcAggregateLoad(EnergyPlusData &state)
         SumQnMonth /= hrsPerMonth;
         this->QnMonthlyAgg(MonthNum) = SumQnMonth;
     }
-    if (this->prevHour != state.dataGroundHeatExchanger->locHourOfDay) {
-        this->prevHour = state.dataGroundHeatExchanger->locHourOfDay;
-    }
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:2435]:(style),[duplicateConditionalAssign],The statement 'if (this->prevHour!=state.dataGroundHeatExchanger->locHourOfDay) this->prevHour=state.dataGroundHeatExchanger->locHourOfDay' is logically equivalent to 'this->prevHour=state.dataGroundHeatExchanger->locHourOfDay'.